Assets/Project/Scripts/MenuSelectScene/LevelSelect/NumClearTreeHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Project.Scripts.Utils.Definitions;
+using Project.Scripts.Utils.PlayerPrefsUtils;
+using System;
+using System.Linq;
+
+namespace Project.Scripts.MenuSelectScene.LevelSelect
+{
+    /// <summary>
+    /// クリアしているステージの割合がクリア条件
+    /// </summary>
+    public class NumClearTreeHandler : IClearTreeHandler
+    {
+        private ETreeId _treeId;
+        private int _clearNumThreshold;
+        private int _stageNum;
```

Comment:
```suggestion
        private readonly int _stageNum;
```

ドキュメンテーションコメントもお願いします